### PR TITLE
Fixing feedback

### PR DESCRIPTION
**Pull request by fionamei**

### CODE REVIEW

File: ComfortZoneApp/app/src/main/java/com/example/comfortzone/InitialComfortActivity.java [hunk 1]
```diff
@@ -110,11 +110,11 @@ public void done(ParseException e) {
...
 
                     if (level == 0) {
-                        addEntry(entryZero);
+                        tracker.addEntry(entryZero);
                     } else if (level == 5) {
-                        addEntry(entryFive);
+                        tracker.addEntry(entryFive);
                     } else if (level == 10) {
-                        addEntry(entryTen);
+                        tracker.addEntry(entryTen);
```
Comment (kimjaspermui) [inline ComfortZoneApp/app/src/main/java/com/example/comfortzone/InitialComfortActivity.java:114]: ohh, nice catch, you already have the tracker, so we should have just add the entry there. However, 2 questions:
1. if you do this, don't you need to to call saveInBackground?
2. why couldn't you add entry to tracker before saveInBackground  one line 105 above?

File: ComfortZoneApp/app/src/main/java/com/example/comfortzone/InitialComfortActivity.java [hunk 2]
```diff
@@ -26,6 +26,7 @@
 public class InitialComfortActivity extends AppCompatActivity {
 
     public static final String TAG = "InitialComfortActivity";
+    public static final int totalLevels = 11;
```
Comment (kimjaspermui) [inline ComfortZoneApp/app/src/main/java/com/example/comfortzone/InitialComfortActivity.java:26]: for constants you should capitalize them: TOTAL_LEVELS

File: ComfortZoneApp/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -5,4 +5,14 @@
...
+    <string name="welcome">Welcome!</string>
+    <string name="introduction">ComfortZone will track your comfort zones for different temperature levels! A \'0\' represents the point where you are crazy cold! Similarly, a \'10\' represents a crazy hot degree. And \'5\' is your estimated perfect temperature!</string>
+    <string name="what_is_your_0">What is your 0?</string>
+    <string name="what_is_your_5">What is your 5?</string>
+    <string name="what_is_your_10">What is your 10?</string>
+    <string name="sign_in">Sign in</string>
+    <string name="don_t_have_an_account">Don\'t have an account?</string>
+    <string name="signup">Signup</string>
+    <string name="hello">HELLO!</string>
+    <string name="your_perfect_temperature_is">Your perfect temperature is:</string>
```
Comment (kimjaspermui) [inline ComfortZoneApp/app/src/main/res/values/strings.xml:17]: really nice!!

**Review by kimjaspermui — APPROVED** [review]
Fiona, really love your changes here! you addressed the feedbacks and made your code so much more organized and also applied all feedback about the commits making them so much easier to review with digestible pieces and clear titles! great work and keep it up!

File: ComfortZoneApp/app/src/main/res/values/strings.xml (hunk 3 above)
Comment (varhalemonica) [reply ComfortZoneApp/app/src/main/res/values/strings.xml:17]: Yes, good to see you incorporating feedback and making the changes. Great job Fiona :)

**Review by varhalemonica — APPROVED** [review]

**Review by kimjaspermui — APPROVED** [review]
fixes look good!

**Pull request merged**